django_ses/views.py

```diff
@@ -537,6 +541,61 @@ def handle_click(self, notification, message):
             message=message
         )
 
+    def handle_received(self, notification, message):
+        mail_obj = message.get('mail')
+        content = message.get('content')
+        receipt = message.get('receipt')
+
+        # Logging
+        logger.info(
+            'Received inbound notification',
+            extra={
+                'notification': notification,
+            },
+        )
+
+        if mail_obj.get('messageId') == 'AMAZON_SES_SETUP_NOTIFICATION':
+            logger.debug('Received AWS SES setup notification, skipping...')
+            return
+
+        try:
+            module, fn = settings.AWS_SES_INBOUND_HANDLER.rsplit('.', 1)
+            module = importlib.import_module(module)
+            if hasattr(module, fn):
+                inbound_handler = getattr(module, fn)
+            else:
+                logger.error(f'Method {fn} not found in module {module}')
+                return
+        except ModuleNotFoundError:
+            logger.error(f'Module {module} could not be imported')
+            return
+
+        try:
+            handler_res = inbound_handler(**{
```

Comment:
Are there any other params in the `message` that we may want to pass to handlers?

Reply:
No, `mail`, `receipt` and `content` (only for the `sns` type of events) are the only keys in the `message`.